LoopOnboardingKitUI/View Controllers/OnboardingUICoordinator.swift

```diff
@@ -272,7 +266,14 @@ class OnboardingUICoordinator: UINavigationController, OnboardingViewController
     }
 
     private func stepFinished() {
-        if let nextScreen = currentScreen.next() {
+        var nextScreen: OnboardingScreen?
+
+        nextScreen = currentScreen.next()
+        if nextScreen == .nightscoutChooser && service?.isOnboarded == true {
```

Comment:
Hm...this special case is a bit odd and not obvious... Consider adding a code comment here as to why.